File: app/src/main/java/com/checkin/app/checkin/session/model/QRResultModel.java
package com.checkin.app.checkin.session.model;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.annotation.JsonProperty;

import androidx.annotation.Nullable;

@JsonIgnoreProperties(ignoreUnknown = true)
public class QRResultModel {
    @JsonProperty("session_pk")
    private long sessionPk;

    @JsonProperty("restaurant_pk")
    private long restaurantPk;

    @JsonProperty("detail")
    private String detail;

    @Nullable
    @JsonProperty("table")
    private String table;

    public QRResultModel() {
    }

    public long getSessionPk() {
        return sessionPk;
    }

    public long getRestaurantPk() {
        return restaurantPk;
    }

    public String getDetail() {
        return detail;
    }

    @Nullable
    public String getTable() {
        return table;
    }
}

File: app/src/main/java/com/checkin/app/checkin/session/models/QRResultModel.kt
package com.checkin.app.checkin.session.models

import com.fasterxml.jackson.annotation.JsonIgnoreProperties
import com.fasterxml.jackson.annotation.JsonProperty

@JsonIgnoreProperties(ignoreUnknown = true)
data class QRResultModel(
        @JsonProperty("session_pk") val sessionPk: Long = 0,
        @JsonProperty("restaurant_pk") val restaurantPk: Long = 0,
        val detail: String? = null,
        val table: String? = null,
        @JsonProperty("is_master_qr") val isMasterQr: Boolean
)